README.md

```diff
@@ -77,7 +76,6 @@ The following variants support ECS:
 
 We also have variants that are designed to be Kubernetes worker nodes in VMware:
 
-* `vmware-k8s-1.20`
```

Comment:
Traditionally we add the removed variants to the "The following variants are no longer supported" section below.